docs/develop/api-reference/index.mdx

```diff
@@ -460,4 +460,14 @@ Collections may contain different types of structures depending on your use case
 * [Lists](./list-collections.md) are a collection of ordered values, sorted in the sequence each element was inserted.
 * [Sets](./set-collections.md) are an unordered collection of unique strings.
 
-For more in-depth information on usage, see [collection data types](./../datatypes.md).
\ No newline at end of file
+For more in-depth information on usage, see [collection data types](./../datatypes.md).
+
+## Current status of API support in SDKs
```

Comment:
Maybe just as TODO but maybe once we add incubating section and mini matrix in each sdk repo we can come back and add note about that for where to look